ui/app/routes/clients.js

```diff
@@ -22,7 +22,8 @@ export default class ClientsRoute extends Route.extend(WithForbiddenState) {
   model() {
     return RSVP.hash({
       nodes: this.store.findAll('node'),
-      agents: this.store.findAll('agent'),
```

Comment:
Note: I don't think this is a needed line for the clients index (anymore?).

As best as I can tell, non-authoritative-region management tokens do _not_ have access to the server members list (cannot do `nomad server members` either)

This hadn't been a concern in the UI before, but the following is a situation where it would be problematic:
- I am a management token for a non-authoritative region. I view the `servers` page and get a "not authorized" message. OK, that's fine, I'm not part of the global/authoritative region.
- I switch to the clients tab, which should at least show me the clients within the region that holds my token. However, since the model for this page has a check for server members (agents in Ember), this page also gives me a "not authorized" error.

This removal makes it so non-authoritative region tokens with `node list` abilities can still view this page.

Reply:
> As best as I can tell, non-authoritative-region management tokens do not have access to the server members list (cannot do nomad server members either)

To close the loop on a Slack thread, the `Status.Members` RPC isn't forwarded between servers (much less regions), so a local management token can get the server members list but only when sending that request to its own region.